src/library/Hooks/useErasToTimeleft/index.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { useApi } from 'contexts/Api';
+
+export const useErasToTimeleft = () => {
+  const { consts } = useApi();
+  const { epochDuration, expectedBlockTime, sessionsPerEra } = consts;
+
+  // converts a number of eras to timeleft in seconds.
+  const erasToTimeLeft = (eras: number) => {
```

Comment:
Rename this function to `getErasToTimeLeft`.

When we are getting something (as opposed to setting something), we can use the `get` prefix.